src/App.tsx

```diff
@@ -28,6 +30,8 @@ function App() {
                 <Route index element={<DashboardPage />} />
                 <Route path={ROUTES.patients} element={<PatientsPage />} />
                 <Route path={ROUTES.doctors} element={<DoctorsPage />} />
+                <Route path="/doctor/:docId" element={<DoctorDetail />} />
```

Comment:
A ten path nie może być w ROUTES zdefiniowany? :P 
unikajmy skrótów, doctorId będzie idealne, docId może podpowiadać nam że to id dokumentu?